### ISSUE TITLE
[Backend] Route for changing password

### ISSUE THREAD

**Issue by jackfrost13** [HITK-TECH-Community/Community-Website#94]
### Work to be done:

- Add a route for changing password. `[PUT]` `/admin/password`
- Validation Schema for the request body
- API Testing using Postman demonstrating all possible cases(valid, invalid) with test cases

#### Request Body Structure:

| Parameter   | Description      |
| ----------- | ---------------- |
| `oldPassword` | string, required  |
| `newPassword` | string, required |

_**NOTE: Please go through the backend code to get a feel of the code structure and follow that.**_

_For API Testing briefing, refer to https://github.com/HITK-TECH-Community/Community-Website/issues/81_

**Comment by jackfrost13** [comment]
@mak-ux Please follow contributing guidelines. You are supposed to work on the issue only once you are assigned.
Moreover, your PR doesn't adhere to this issue. 
Feel free to ask if you have any queries :)

**Comment by samir-0711** [comment]
I want to work on this. Can you assign me this issue?

**Comment by mak-ux** [comment]
sorry for that, actually this is my first time working on open source so sorry.I thought we can work directly looking at the issue but now I got It.

**Comment by mak-ux** [comment]
Sir here I want to add something that in req. body you are asking for the old password and then changing to a new password, In case Admin forgot his password then he will not be able to change or even login so we must have a reset password link so that from there on can try for changing the password.

**Comment by jackfrost13** [comment]
> sorry for that, actually this is my first time working on open source so sorry.I thought we can work directly looking at the issue but now I got It.

No issues, just keep in mind whenever you are making open source contributions, you go through the contribution guidelines and the code of conduct ;)

**Comment by jackfrost13** [comment]
> Sir here I want to add something that in req. body you are asking for the old password and then changing to a new password, In case Admin forgot his password then he will not be able to change or even login so we must have a reset password link so that from there on can try for changing the password.

Firstly no need for the "Sir" salutation, feel free to refer anyone by their names or tagging them.

Yes as this route is meant only for change password and not for forget password.
I cannot merge your code for this issue, I will reference a new issue and then you can contribute to that. Sounds fair? 


Please go through the code structure that is followed. Your code doesn't follow the code style and structure. We can discuss more on this once I open the new issue.

**Comment by Schitiz** [comment]
Hey @jackfrost13 I would like to work on this issue.

**Comment by jackfrost13** [comment]
Sure, go ahead. Happy Coding.
Make sure you follow the codebase structure.

**Comment by N00Bmaster2810** [comment]
Hi @jackfrost13 ,  if this issue is open can I work on it under SWOC?

**Comment by jackfrost13** [comment]
@Schitiz Please update your progress on the issue ASAP!

**Comment by jackfrost13** [comment]
@Schitiz It been considerably sufficient time for completing the issue. Please update your progress else we will be left with nothing but to assign someone else on this issue

**Comment by N00Bmaster2810** [comment]
Hey @jackfrost13 can I work on this?

**Comment by N00Bmaster2810** [comment]
Thanks @jackfrost13
